x-pack/solutions/security/plugins/security_solution/public/detection_engine/rule_management_ui/components/rules_table/upgrade_prebuilt_rules_table/translations.tsx

```diff
@@ -86,22 +86,22 @@ export const RULE_TYPE_CHANGE_CALLOUT_TITLE = i18n.translate(
 export const RULE_TYPE_CHANGE_CALLOUT_DESCRIPTION = i18n.translate(
   'xpack.securitySolution.detectionEngine.upgradeRules.ruleTypeChangeCalloutDescription',
   {
-    defaultMessage: 'Elastic update has rule type changed.',
+    defaultMessage: "The rule's type will change after you update this rule.",
```

Comment:
Some small tweaks here too. Let me know if they're ok.

```suggestion
    defaultMessage: "The rule type will change if you update this rule.",
```